src/test/java/uk/gov/hmcts/reform/sscs/service/HearingsServiceTest.java

```diff
@@ -62,12 +62,7 @@
 @ExtendWith(MockitoExtension.class)
 class HearingsServiceTest {
     private static final long HEARING_REQUEST_ID = 12345;
-    private static final String HMC_STATUS = "TestStatus";
```

Comment:
were these removed as part of a refactor?

Reply:
They were unused variables, same with others